### DISCUSSION TITLE
Adobe Air on Big Sur

### DISCUSSION THREAD

**Discussion by G-Powder** [airsdk/Adobe-Runtime-Support#630] [Q&A]
Hello,

Has anyone had an issue installing the AIR runtime on Big Sur?  We recently attempted to install it for a few clients from our usual source located here:  https://get.adobe.com/air/

It continuously fails.
Any insight would be much appreciated.

Thanks.

**Comment by ajwfrost** [comment]
Yes I think there is a problem in the Adobe version 32 which will cause a problem with Big Sur. This should be fixed in our latest release of the SDK v33 though - but we don't have a shared runtime so the applications would need to be re-bundled with the captive runtime mechanism for this.

thanks

**Comment by G-Powder** [comment]
Got it.  Thanks very much for the quick response.

**Comment by NicolasDionB** [comment]
I have reports of camera device not being available in my AIR app on Mac OS Big Sur, even using the latest SDK 33.1.1.300. Anyone else experienced this?

**Comment by apofis1969** [comment]
@NicolasDionB we have the same problem with big sur. Camera permissions are set but the camera keeps deactivated on big sur. No matter which SDK. SDK 33.1.1.300 creates a lot of bugs with macOS Catalina and crashes with Big Sur.

**Comment by NicolasDionB** [comment]
Also follow issue #532 for camera problems on Big Sur.

**Comment by spamshaker** [comment]
Hi
where can I get 33.x SDK to try with our issue?
https://www.adobe.com/devnet/air/air-sdk-download.html still 32
https://labs.adobe.com/downloads/ also 32

**Comment by spamshaker** [comment]
> Hi
> where can I get 33.x SDK to try with our issue?
> https://www.adobe.com/devnet/air/air-sdk-download.html still 32
> https://labs.adobe.com/downloads/ also 32

never mind already googled ;)
https://airsdk.harman.com/download

**Comment by NicolasDionB** [comment]
A quick note: I checked on MacOs 11.0.1 and still have the issue. I tried activating the camera by calling `Camera.getCamera()` with no parameters and it returned `null`. I also Googled to try to find if there was entitlements or info.plist addition to make but found nothing. Also tried to re-notorize the app with no success. Really hope we can nail this one, it's kind of major!
